mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -106,10 +107,14 @@ def setup_and_check(self):  # {{{
         regions = [regions[index] for index in regionIndicesToPlot]
 
         for region in regions:
-            filePrefix = 'ohc_{}_{}.png'.format(region, mainRunName)
-            self.xmlFileNames.append('{}/{}.xml'.format(self.plotsDirectory,
-                                                        filePrefix))
-            self.filePrefixes[region] = filePrefix
+            if plotOriginalFields:
```

Comment:
`plotOriginalFields` is undefined at this point, so it should be defined first:
```
plotOriginalFields = config.getboolean(configSectionName,
 +                                               'plotOriginalFields')
```

Reply:
configSectionName is also undefined and will need to either be defined or replaced with 'timeSeriesOHC'.  Please run a quick test to make sure things are working before pushing commits.

Reply:
Also, I would move this if/else outside of the for loop on regions.  It doesn't really matter but there's no reason to do it once per region...